src/Editor.cpp

```diff
@@ -610,7 +610,12 @@ void Editor::draw_inspector(std::shared_ptr<EditorWindow> const& window)
     }
 
     glm::vec3 scale = entity->transform->get_local_scale();
+    glm::vec3 cached_scale = scale;
```

Comment:
Can we rename this to `old_scale` and `old_value` in the function? "Cache" word is, most of the time, intended for things that are cached for use in later frames or invocations of the same function